test/help/sexpr-wasm.txt

```diff
@@ -22,14 +22,14 @@ examples:
   $ wast2wasm spec-test.wast --spec -o spec-test.json
 
 options:
-  -v, --verbose                        use multiple times for more info
-  -h, --help                           print this help message
-  -d, --dump-module                    print a hexdump of the module to stdout
-  -o, --output=FILE                    output file for the generated binary format
-      --spec                           parse a file with multiple modules and assertions, like the spec tests
-      --use-libc-allocator             use malloc, free, etc. instead of stack allocator
-      --no-canonicalize-leb128s        Write all LEB128 sizes as 5-bytes instead of their minimal size
-      --debug-names                    Write debug names to the generated binary file
-      --no-check                       Don't check for invalid modules
-      --no-check-assert-invalid        Don't run the assert_invalid checks
+  -v, --verbose                                      use multiple times for more info
+  -h, --help                                         print this help message
+  -d, --dump-module                                  print a hexdump of the module to stdout
+  -o, --output=FILE                                  output file for the generated binary format
+      --spec                                         parse a file with multiple modules and assertions, like the spec tests
+      --use-libc-allocator                           use malloc, free, etc. instead of stack allocator
+      --no-canonicalize-leb128s                      Write all LEB128 sizes as 5-bytes instead of their minimal size
+      --debug-names                                  Write debug names to the generated binary file
+      --no-check                                     Don't check for invalid modules
+      --no-check-assert-invalid-and-malformed        Don't run the assert_invalid or assert_malformed checks
```

Comment:
yeah, but like getopt_long you can abbreviate arguments as long as they're not ambiguous, so `--no-check-assert` (or even `no-check-`) works too.